data/filedefs/filetypes.common

```diff
@@ -129,14 +129,15 @@ comment_line_doc=comment_doc
 comment_doc_keyword=comment_doc,bold
 comment_doc_keyword_error=comment_doc,italic
 
-number=0x007f00
+number=0x808000
```

Comment:
Sorry I didn't remember that I did that, but the `number` color 0x007f00 is very similar to `string_2`, which is very confusing for Julia.

Reply:
But since this is the default colour scheme you are changing it for everybody and there have been no complaints that I could find.  

If its really a problem for Julia (why for Julia specifically and not other languages that use `string_2` ?) maybe map Julia numbers to something other than "number" in `highlighting_mappings.h`, "julia_number" for eg, so you can have your own colour.  Adding new highlight type names won't modify existing ones.

Reply:
ok, then I can create a `number_3` color for julia numbers.